policy_test.go

```diff
@@ -100,6 +100,24 @@ func TestGetPolicyRevision(t *testing.T) {
 		t.Error(err)
 	}
 
+	if data.Name != policyName {
+		t.Errorf("Unexpected policy name: %+v", data.Name)
+	}
+
+	if data.RevisionID != policyRevision {
+		t.Errorf("Unexpected policy revision ID: %+v", data.RevisionID)
+	}
+
+	if data.RunList[0] != "recipe[base::default]" {
+		t.Errorf("Unexpected policy run list: %+v", data.RevisionID)
+	}
+
+	if val, ok := data.NamedRunList["os"]; !ok {
+		t.Error("Expected os NamedRunList policy to be present in the policy information")
+	} else if val[0] != "recipe[hardening::default]" {
+		t.Error("Expected named run list for the policy, got: ", val[0])
+	}
+
```

Comment:
On closer inspection - can you add validation for the `included_policy_locks` field being parsed?